compiler/noirc_evaluator/src/errors.rs

```diff
@@ -7,7 +7,7 @@
 //! An Error of the former is a user Error
 //!
 //! An Error of the latter is an error in the implementation of the compiler
-use acvm::FieldElement;
+use acvm::{acir::InvalidInputBitSize, FieldElement};
```

Comment:
I still would like for us to not leak this ACVM solving error into the noirc_evaluator errors. We should have an error that looks similar to `UnsupportedIntegerSize` with two `u32` values representing the bit sizes, and a name like `InvalidBlackBoxInputBitSize`.